webapp/src/websocket/index.ts

```diff
@@ -14,3 +14,9 @@ export function handleDisconnect(store: Store<GlobalState, Action<Record<string,
         store.dispatch(toggleIsConnected(false) as Action);
     };
 }
+
+export function handleSubscriptionDeleted(store: Store<GlobalState, Action<Record<string, unknown>>>) {
+    return (_: WebsocketEventParams) => {
```

Comment:
When this websocket event is received here, are you refetching the subscriptions from the API every time? You should refetch only when the RHS is open and it contains the Azure Devops plugin.
See this: https://github.com/Brightscout/mattermost-plugin-servicenow/blob/9dc472a83c62dd840e462fb7743bdf88efd9577c/webapp/src/websocket/index.ts#L56

Reply:
even when websocket is fired we are fetching this only when `ProjectDetails` component is mounted as the API call is triggered from this component only

Reply:
Suppose a subscription is deleted and a websocket event was fired but at that time, the RHS was not open. So, now when we open the RHS, make sure the API doesn't get called twice. This was happening in ServiceNow. Here, the code structure is different, but still we should test it.

Reply:
tested, and fixed this